prow/prowjobs/GoogleCloudPlatform/gcsfuse/gcsfuse-pytorch-dino-periodic.yaml

```diff
@@ -0,0 +1,37 @@
+periodics:
+- name: gcsfuse-pytorch-dino-periodic
+  cluster: gcsfuse-prow-test
+  interval: 180h
+  decorate: true
+  annotations:
+    testgrid-dashboards: gcsfuse
+    testgrid-tab-name: pytorch-dino-periodic
+    description: "Runs pytorch dino model every 7 days using gcsfuse."
+    spec:
```

Comment:
I think you have a bad indent here; your entire spec is not itself an "annotation". That would explain the unmarshalling errors you're getting

Reply:
Thanks, fixed that.